documentation/modules/auxiliary/scanner/http/http_hsts.md

```diff
@@ -0,0 +1,66 @@
+## Vulnerable Application
+
+### Description
+
+This module checks to see whether or not the scanned systems return the HSTS header to enforce HSTS.
+
```

Comment:
did you set the server up yourself? if so, can you provide instructions on how to do so on apache/nginx (just one is fine).

Reply:
Sorry didn't set the server up myself, but I'll look into spinning up an Apache instance with this enabled.

Reply:
Got the Apache server set up, next commit should add instructions on how to do so.